vtm.js

```diff
@@ -0,0 +1,339 @@
+          abilities: null,
+          blood: ">0"
+        },
+        nosferatu:{
+          display: "Nosferatu",
+          abilities: ["obfuscate", "potency", "animalism"],
+          blood: ">0"
+        },
+      },
+      bio: {
+        name : null,
+        age : null,
+        clanSelect: null,
+        bloodPotency : 0,
+        skillPrioritySelect: null,
+      },
+      //TODO^^DATA ABOVE IS WIP^^
+      attributes: {
+          display: "Attributes",
+          parent: "attributes",
+          data: [
+            {
+                display: "Physical",                 
+                list: [
+                    { 
+                      id: "strength",
+                      display: "Strength",
+                      description: "",
+                      value: 1 
+                    },{ 
+                      id: "dexterity", 
+                      display: "Dexterity",
+                      description: "", 
+                      value: 1 
+                    },{ 
+                      id: "stamina", 
+                      display: "Stamina",
+                      description: "",
+                      value: 1 
+                    },
+                ],
+            },{
+                display: "Social",                 
+                list: [
+                    { 
+                      id: "charisma", 
+                      display: "Charisma",
+                      description: "",
+                      value: 1 
+                    },{ 
+                      id: "manipulation", 
+                      display: "Manipulation", 
+                      description: "",
+                      value: 1 
+                    },{ 
+                      id: "composure", 
+                      display: "Composure", 
+                      description: "",
+                      value: 1 
+                    }     
+                ],
+            },{
+                display: "Mental",                 
+                list: [
+                    { 
+                      id: "intelligence",
+                      display: "Intelligence",
+                      description: "",
+                      value: 1 
+                    },{ 
+                      id: "wits", 
+                      display: "Wits",
+                      description: "",
+                      value: 1
+                    },{ 
+                      id: "resolve", 
+                      display: "Resolve",
+                      description: "",
+                      value: 1                      
+                    } 
+                ],
+        }],
+        resource: [0, 1, 4, 3, 1, 0],/*vtm5e attribute distribution: 1 times 4 dots; 3 times 3 dots, 4 times 2 dots
+        sum of values in resource <= amount of values in section*/
+        //scale: [1, 5], // or maybe scale (range whatever) is calculated as resource.index of -1 and resource.length? (which would be 6 so no... so i guess len-1?)       
```

Comment:
scale sa dalo prec, tak koment moze ist tiez

Reply:
hej to som tam nechal len kvoli tej myslienke aby to aj peter videl